node/core/dispute-coordinator/src/metrics.rs

```diff
@@ -221,12 +221,12 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
-			priority_queue_size: prometheus::register(
+			participation_priority_queue_size: prometheus::register(
 				prometheus::Gauge::new("polkadot_parachain_dispute_priority_queue_size", 
```

Comment:
nit: Should the metric names be updated to match the new variable names? e.g. `"polkadot_parachain_dispute_participation_priority_queue_size"`